### DISCUSSION TITLE
Enhancement: Taunt

### DISCUSSION THREAD

**Discussion by SoffiMeower** [FunkinCrew/Funkin#1595] [Ideas]
Ever wanted to go along with the song as the enemy is singing, or wanted to see Boyfriend do his cute peace sign on another week besides from week one? Well, I believe a taunt button is in order! Where if you press Q or E (Or another key if the programmer chooses so.), it'll have them do their peace sign before going back to their regular head bobbing animation.

**Comment by LoafyToast** [comment]
I would honestly love this
